csrankings-c.csv

```diff
@@ -1067,6 +1067,7 @@ Chu-Xu Zhang,Brandeis University,https://chuxuzhang.github.io,ZbZ0l0oAAAAJ
 Chuadhry Mujeeb Ahmed,University of Strathclyde,https://pureportal.strath.ac.uk/en/persons/chuadhry-mujeeb-ahmed/publications,l_qJD3IAAAAJ
 Chuan Chen 0001,Sun Yat-sen University,http://106.52.19.28/faculty/index.html,yJId-nEAAAAJ
 Chuan Lu,Aberystwyth University,https://www.aber.ac.uk/en/cs/staff-profiles/listing/profile/cul,tvY6seQAAAAJ
+Chuan Luo 0002,Beihang University,https://soft.buaa.edu.cn/teachershouw.jsp?urltype=news.NewsContentUrl&wbtreeid=1262&wbnewsid=9648,MfJGmUUAAAAJ

```

Comment:
URL is a 403 error. Please provide a direct home page for this faculty member.

Reply:
The homepage for this faculty member (https://shi.buaa.edu.cn/chuanluo/en/index.htm) has been updated as requested. I hope that this update could address your comment. Thanks!